internal/web/routes/routes.go

```diff
@@ -0,0 +1,18 @@
+package routes
+
+import (
+	"context"
+
```

Comment:
Consider validating context to ensure it's still active before using it within the controller methods to avoid context-related errors later on in processing requests.